### DISCUSSION TITLE
Unable to submit a new request to student pack

### DISCUSSION THREAD

**Discussion by romeoahmed** [community/community#65407] [GitHub Education] [answered]
### Hi everyone,

My github student pack **expired** a month ago, but I didn't see it, now when I want to resubmit the application, it says **"There are outstanding issues with your GitHub account. Please contact GitHub Support for more information."**
I have submitted a ticket, but **no response**.
I know I am an eligible student, what should I do?

**Comment by 2511587947** [comment]
I had a similar problem

**Reply by romeoahmed** [reply]
have you solve this problem?

**Reply by 2511587947** [reply]
Still not solved

**Comment by romeoahmed** [comment]
It's been four days since I submitted a ticket, but no response from Github. 😩
Have you ever encountered this situation?

**Comment by romeoahmed** [comment, marked as answer]
Well, after almost two months, GitHub solved my problem.
So what we can do for this kind of problem is to submit a ticket to Github Support and wait.😂
